--- APromotesMachineClause.java	2012-05-24 07:07:28.000000000 +0100
+++ APromotesMachineClause.java	2012-05-24 07:10:14.000000000 +0100
@@ -35,6 +35,12 @@
         ((Analysis) sw).caseAPromotesMachineClause(this);
     }
 
+    public LinkedList<PExpression> getOperationNames()
+    {
+    	return this._machineNames_;
+    }
+    
+    @Deprecated
     public LinkedList<PExpression> getMachineNames()
     {
         return this._machineNames_;
